spec/models/game_spec.rb

```diff
@@ -24,6 +24,29 @@
     end
   end
 
+  describe '#state' do
+    context 'when absent' do
+      let(:game) { build(:game, state: nil) }
+
+      it 'is invalid' do
+        expect(game.valid?).to be false
+      end
+
+      it 'raises a database error' do
+        expect { game.save(validate: false) }
+          .to raise_error ActiveRecord::NotNullViolation
+      end
+    end
+
+    context 'when present' do
+      let(:game) { build(:game, state: 1) }
```

Comment:
Same thing here we want to use enum cases in our code. It's more readable than a magic number
```rb
let(:game) { build(:game, state: :pending) }
```